packages/ui/src/bounty/modals/SubmitWorkModal/SubmitWorkModal.tsx

```diff
@@ -116,63 +123,65 @@ export const SubmitWorkModal = () => {
-                >
-                  <InputText
-                    id="field-work-title"
-                    value={state.context.workTitle}
-                    onChange={(e) => send('SET_WORK_TITLE', { workTitle: e.target.value })}
-                    placeholder={t('modals.submitWork.submitWorkInput.workTitlePlaceholder')}
+      <ScrolledModalBody>
+        <ScrolledModalContainer>
+          {state.matches(SubmitWorkStates.generalParameters) && (
+            <RowGapBlock gap={24}>
+              <Row>
+                <RowGapBlock gap={8}>
+                  <h4>{t('modals.submitWork.subtitle')}</h4>
+                </RowGapBlock>
+              </Row>
+              <Container
+                disabled
+                label="Bounty ID"
+                tooltipText={t('modals.submitWork.submitWorkInput.bountyId')}
+                inputSize="l"
+              >
+                <TextBig value bold>
+                  {modalData.bounty.id}
+                </TextBig>
+              </Container>
+              <Row>
+                <RowGapBlock gap={20}>
+                  <SelectedMember
+                    disabled
+                    member={activeMember}
+                    label={t('modals.submitWork.submitWorkInput.memberId')}
                   />
-                </InputComponent>
-                <InputComponent
-                  id="field-description"
-                  label={t('modals.submitWork.submitWorkInput.entryDescription')}
-                  required
-                >
-                  <CKEditor
+                  <InputComponent
+                    id="field-work-title"
+                    required
+                    inputSize="m"
+                    label={t('modals.submitWork.submitWorkInput.workTitle')}
+                    message={workTitleValidation}
+                    validation={state.context.workTitle.length > 70 ? 'invalid' : undefined}
+                  >
+                    <InputText
+                      id="field-work-title"
+                      value={state.context.workTitle}
+                      onChange={(e) => send('SET_WORK_TITLE', { workTitle: e.target.value })}
+                      placeholder={t('modals.submitWork.submitWorkInput.workTitlePlaceholder')}
+                    />
+                  </InputComponent>
+                  <InputComponent
                     id="field-description"
-                    minRows={3}
-                    onChange={(event, editor) => send('SET_WORK_DESCRIPTION', { workDescription: editor.getData() })}
-                    onReady={(editor) => editor.setData(state.context.workDescription || '')}
-                  />
-                </InputComponent>
-              </RowGapBlock>
-            </Row>
-          </RowGapBlock>
-        )}
-      </ModalBody>
+                    label={t('modals.submitWork.submitWorkInput.entryDescription')}
+                    required
```

Comment:
You need to set "inputSize" to "auto" to prevent an outline issue with CKEditor.
```suggestion
                    inputSize="auto"
                    required
```